ui/apps/platform/src/Containers/Clusters/Components/SensorStatus.tsx

```diff
@@ -1,60 +1,45 @@
 import React, { ReactElement } from 'react';
-import { Tooltip } from '@patternfly/react-core';
 
-import { getDateTime, getDistanceStrict } from 'utils/dateUtils';
-import HealthStatus from './HealthStatus';
+import { ClusterHealthStatus } from 'types/cluster.proto';
+import { getDistanceStrict } from 'utils/dateUtils';
+
 import { healthStatusStyles, isDelayedSensorHealthStatus } from '../cluster.helpers';
-import { ClusterHealthStatus } from '../clusterTypes';
+
 import HealthLabelWithDelayed from './HealthLabelWithDelayed';
+import HealthStatus from './HealthStatus';
 import HealthStatusNotApplicable from './HealthStatusNotApplicable';
 
-/*
- * Sensor Status in Clusters list or Cluster side panel
- *
- * Caller is responsible for optional chaining in case healthStatus is null.
- */
-
 type SensorStatusProps = {
     healthStatus: ClusterHealthStatus;
-    isList?: boolean;
 };
 
-function SensorStatus({ healthStatus, isList = false }: SensorStatusProps): ReactElement {
+function SensorStatus({ healthStatus }: SensorStatusProps): ReactElement {
     if (!healthStatus?.sensorHealthStatus) {
-        return <HealthStatusNotApplicable testId="sensorStatus" isList={isList} />;
+        return <HealthStatusNotApplicable testId="sensorStatus" isList />;
     }
 
     const { sensorHealthStatus, lastContact } = healthStatus;
-    const { Icon, fgColor } = healthStatusStyles[sensorHealthStatus];
-    const currentDatetime = new Date();
 
     const isDelayed = !!(lastContact && isDelayedSensorHealthStatus(sensorHealthStatus));
-    const delayedText = `for ${getDistanceStrict(lastContact, currentDatetime, {
-        partialMethod: 'floor',
-    })}`;
-    const icon = <Icon className={`${isList ? 'inline' : ''} h-4 w-4`} />;
-    const sensorStatus = (
-        <HealthStatus icon={icon} iconColor={fgColor} isList={isList}>
-            <HealthLabelWithDelayed
-                clusterHealthItem="sensor"
-                clusterHealthItemStatus={sensorHealthStatus}
-                isList={isList}
-                isDelayed={isDelayed}
-                delayedText={delayedText}
-            />
-        </HealthStatus>
+    const delayedText = isDelayed ? `(${getDistanceStrict(lastContact, new Date())} ago)` : '';
```

Comment:
Observing for your awareness in case it ever became important in the future, although it does not seem to cause a rendering problem here, something that I have done somewhere (but apparently not here) is store `currentDatetime` in state hook from `new Date()` call in `useEffect` or in case of clusters table, `useInterval` hook, and then pass as prop so that difference in component is pure computation, instead of hidden side effect.